src/Middleware/HttpOverrides/src/ForwardedHeadersExtensions.cs

```diff
@@ -7,15 +7,22 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// Extension methods for enabling <see cref="ForwardedHeadersMiddleware"/>.
+    /// </summary>
     public static class ForwardedHeadersExtensions
     {
         private const string ForwardedHeadersAdded = "ForwardedHeadersAdded";
 
         /// <summary>
-        /// Forwards proxied headers onto current request
+        /// Forwards proxied headers onto current request.
```

Comment:
I tweaked this file a bit. Can you have a look?

Reply:
```suggestion
        /// Applies forwarded headers to their matching fields on the current request.
```